apps/web/playwright/fixtures/workflows.ts

```diff
@@ -56,6 +56,8 @@ export function createWorkflowPageFixture(page: Page) {
   };
 
   const fillNameInput = async (name: string) => {
+    // eslint-disable-next-line playwright/no-wait-for-timeout
+    await page.waitForTimeout(300);
```

Comment:
And here. Now that I think about it. This could be a symptom of flickering since the test might be finding the "workflow-name" selector, then a flicker happens and the fill fails. This could be solved by using proper skeletons or finding the reason for the flicker.

Reply:
yeah, true, let me revert these changes to e2e tests